### PR TITLE
feat(mon): create balance-mon service

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Creates a new balance-mon service inside of chain-mon and removes the legacy balance-monitor that relied on Forta. New balance-mon is much simpler and achieves the same results that the legacy package was achieving.

### CODE REVIEW

File: .github/workflows/release.yml
Comment (maurelian) [inline .github/workflows/release.yml:1]: comment only: could GH have made this diff any less readable?

File: packages/chain-mon/.env.example [hunk 1]
```diff
@@ -1,3 +1,13 @@
+###############################################################################
+#                               ↓ balance-mon ↓                               #
+###############################################################################
+
+# RPC pointing to network to monitor balances on
+BALANCE_MON__RPC=
+
+# Comma-separated list of addresses to monitor
+BALANCE_MON__ADDRESSES=
```
Comment (maurelian) [inline packages/chain-mon/.env.example:9]: I'm not sure the best way to do this in the `BaseServiceV2` arch, but I think it'd be preferable if we could put names on the addresses like with the previous balance monitor. That would make it much easier to configure dashboards and alerting on multiple network.

https://github.com/ethereum-optimism/optimism/blob/ef5c1f7737cc272a1eb8d880de3bfe1563da9f95/packages/balance-monitor/.env.example#L6-L12

Comment (smartcontracts) [reply packages/chain-mon/.env.example:9]: Easiest options here are comma separated or JSON array, either works

Comment (smartcontracts) [reply packages/chain-mon/.env.example:9]: JSON array is slightly easier to read imo

Comment (smartcontracts) [reply packages/chain-mon/.env.example:9]: Easy to configure:

```
BALANCE_MON__ACCOUNTS='[
  {
    "address": "0x68108902De3A5031197a6eB3b74b3b033e8E8e4d",
    "nickname": "wallet1"
  }
]'
```

**Review by maurelian — APPROVED** [review]

**Comment by maurelian** [comment]
Wondering if ArgoCD will pick up the deletion of the `balance-monitor` and deactivate it or something? 
I'd want to have this service deployed before we shut that down.

**Comment by smartcontracts** [comment]
I'm planning to close this PR and replace it with two distinct PRs, one to add the new service and one to remove the old service.

**Pull request closed without merging**